federation/pkg/kubefed/init/init.go

```diff
@@ -229,7 +230,25 @@ func initFederation(cmdOut io.Writer, config util.AdminConfig, cmd *cobra.Comman
 		return err
 	}
 
+	//to talk to federation ctrl plane
```

Comment:
sure madhu; will update with mode elaborate comment, and will take care of this in future also.